src/components/common/TmBalance.vue

```diff
@@ -351,7 +351,7 @@ export default {
     },
     setPreferredCurrency() {
       localStorage.setItem(`preferredCurrency`, this.selectedFiatCurrency)
-      this.preferredCurrency = localStorage.getItem(`preferredCurrency`)
+      this.preferredCurrency = this.selectedFiatCurrency
```

Comment:
OK, I get it. Yes. Like this we avoid retrieving the info from the localStorage at all. Makes so much sense